sprout/appliances/tasks.py

```diff
@@ -305,6 +306,9 @@ def poke_trackerbot(self):
                         name=template_name, preconfigured=False, date=date, custom_data=custom_data,
                         version=template_version, ready=True, exists=True, usable=True)
                     tpl.save()
+                    if provider.type == 'openshift':
```

Comment:
Is this the Sprout provider model? In that case it would be the `.provider_type`.

Reply:
fixed